packages/cfd/src/Containers/cfd-password-modal/modal-elements/password-modal-header.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import { getCFDPlatformLabel } from '@deriv/shared';
+
+type TPasswordModalHeaderProps = {
+    should_set_trading_password: boolean;
+    is_password_reset_error: boolean;
+    platform: string;
+};
+
+const PasswordModalHeader = observer(
+    ({ should_set_trading_password, is_password_reset_error, platform }: TPasswordModalHeaderProps) => {
+        const { ui } = useStore();
+        const { is_mobile } = ui;
+        const element = is_mobile ? 'p' : 'span';
```

Comment:
we are using these values just once in the file, while you make a variable for them and didn't placed them inline?

Reply:
It was like that previously I'll change them and make it inline